snapcraft/snapcraft.yaml

```diff
@@ -15,6 +15,12 @@ apps:
     command: python3
   shell:
     command: bash
+  juju:
+    command: juju
+  lxd:
+    command: lxd
```

Comment:
Unless I'm missing something, I think we probably don't need to include 'lxd', since users shouldn't need to run the 'lxd' program.

I was wondering if it was necessary because part of the container creation process involves calling 'lxd forkstart' but since the binary is renamed to conjure-up.lxd anyway, having it unexported doesn't affect that